particles/Tutorial/Kotlin/Demo/README.md

```diff
@@ -385,3 +385,253 @@ and using the DevTools can see our handles update accordingly.
 Next, we'll implement our players so you can actually play the
 game!
 
+## Creating the Human Player Layer
+
+Currently you should have a tic-tac-toe board that does nothing.
+Sure, you can click it and see the Events populate, but that is
+still rather boring. Let's make it more fun by adding a human 
+player. By now, that original design we made in the first section
+is probably not in the forefront of your mind, so we'll start by
+looking at our design diagram.
+
+![Tic Tac Toe Design](diagrams/TTT.jpg)  
+
+From this, we can see that the Human Player needs a Player and
+Move handle, and these handles connect the Game and Human Player.
+We also need to populate the information about the player, so
+we create a store. This updates our Arcs Manifest File to 
+become:
+
+```
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+
+schema Person
+  name: Text
+  avatar: Text
+
+schema GameState
+  board: Text
+  currentPlayer: Number
+
+schema Event
+  type: Text
+  move: Number
+  time: Number
+
+schema Move
+  move: Number
+
+resource HumanDefault
+  start
+  [
+    {"name": "Human", "avatar": "X"}
+  ]
+
+store HumanStore of Person in HumanDefault
+
+particle TTTBoard in 'TTTBoard.wasm'
+  events: writes [Event]
+  gameState: reads GameState
+  boardSlot: consumes
+
+particle TTTGame in 'TTTGame.wasm'
+  gameState: reads writes GameState
+  playerOne: reads writes Person
+  playerOneMove: reads writes Move
+  events: reads writes [Event]
+  root: consumes
+    boardSlot: provides
+
+particle TTTHumanPlayer in 'TTTHumanPlayer.wasm'
+  gameState: reads GameState
+  events: reads [Event]
+  player: reads Person
+  myMove: writes Move
+
+recipe GamePlayersDemoRecipe
+  playerOne: use HumanStore
+  playerOneMove: create #humanMove
+  TTTGame
+    gameState: reads writes gameState
+    playerOne: reads writes playerOne
+    playerOneMove: reads writes playerOneMove
+    events: reads writes events
+    root: consumes
+      boardSlot: provides board
+  TTTBoard
+    gameState: reads gameState
+    events: writes events
+    boardSlot: consumes board
+  TTTHumanPlayer
+      gameState: reads gameState
+      events: reads events
+      myMove: writes playerOneMove
+      player: reads playerOne
+  description `Kotlin Tutorial TicTacToe Demo`
+
+```
```

Comment:
Instead of quoting huge blocks of code, could you just link to the file itself? I'm worried that over time these code snippets will become out of date, and updating them could be quite tricky.

Probably small code snippets are fine, but this is pretty long, so the benefits of seeing them inline are reduced, and the risks of them going out of date are increased.